.github/workflows/publish.yml

```diff
@@ -1,104 +1,32 @@
-      # Tags that start with the "v" prefix are published on PyPI.
-            - v*
+  push:
+    tags:
+      # After vMajor.Minor.Patch _anything_ is allowed (without "/") !
+      - v[0-9]+.[0-9]+.[0-9]+*
 
 jobs:
-
-    build:
-
-        runs-on: ubuntu-latest
-
-        steps:
-
-            - uses: actions/checkout@v2
-
-            - name: Set up Python 3.10
-              uses: actions/setup-python@v2
-              with:
-                  python-version: '3.10'
-
-            - name: Install pypa/build
-              run: python -m pip install build
-
-            - name: Build a binary wheel and a source tarball
-              run: >-
-                  python -m
-                  build
-                  --sdist
-                  --wheel
-                  --outdir dist/
-            - name: Upload distribution artifact
-              uses: actions/upload-artifact@v2
-              with:
-                  name: release
-                  path: dist/
-
-
-    publish-test:
-
-        name: Build and publish on TestPyPI
-        if: startsWith(github.ref, 'refs/heads/release/')
-        needs: [build]
-        runs-on: ubuntu-latest
-
-        environment:
-            name: Test PyPI
-            url: https://test.pypi.org/project/aiida-cp2k/
-
-        steps:
-            - uses: actions/download-artifact@v2
-              name: Download distribution artifact
-              with:
-                  name: release
-                  path: dist/
-
-            - name: Publish distribution on Test PyPI
-              uses: pypa/gh-action-pypi-publish@release/v1
-              if: startsWith(github.ref, 'refs/heads/release/')
-              with:
-                  user: __token__
-                  password: ${{ secrets.PYPI_API_TOKEN }}
-                  repository_url: https://test.pypi.org/legacy/
-                  skip_existing: true
-
-    publish:
-
-        name: Build and publish on PyPI
-        if: startsWith(github.ref, 'refs/tags')
-        needs: [build]
-        runs-on: ubuntu-latest
-
-        environment:
-            name: PyPI
-            url: https://pypi.org/project/aiida-cp2k/
-
-        steps:
-
-            - uses: actions/download-artifact@v2
-              name: Download distribution artifact
-              with:
-                  name: release
-                  path: dist/
-
-            - uses: softprops/action-gh-release@v0.1.14
-              name: Create release
-              if: startsWith(github.ref, 'refs/tags/v')
-              with:
-                  files: |
-                      dist/*
-                  generate_release_notes: true
-
-            - name: Publish distribution on PyPI
-              uses: pypa/gh-action-pypi-publish@release/v1
-              with:
-                  user: __token__
-                  password: ${{ secrets.PYPI_API_TOKEN }}
+  publish:
+    runs-on: ubuntu-latest
+    if: github.repository == 'aiidateam/aiida-core' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
```suggestion
    if: github.repository == 'aiidateam/aiida-cp2k' && startsWith(github.ref, 'refs/tags/v')
```

Reply:
Also, why the `startsWith`? The `on: push: tags` should already take care of this filtering.

Reply:
fixed.